backend/src/main/java/me/quizzl/backend/models/Question.java

```diff
@@ -53,6 +53,16 @@ public void setCorrectAnswer(String correctAnswer) {
     public void setQuestion(String questionStr) {
         this.questionStr = questionStr;
     }
+    public Boolean evaluateAnswer(Answer answer) {
+       String submittedAnswer = answer.getSubmittedAnswer();
+        
+       if(submittedAnswer.equals(this.correctAnswer)){
```

Comment:
you could probably save yourself the trouble here and just `return submittedAnswer.equals(this.correctAnswer);`